lib/open_api_spex/plug/validate.ex

```diff
@@ -53,18 +53,22 @@ defmodule OpenApiSpex.Plug.Validate do
   alias Plug.Conn
 
   @impl Plug
+  @deprecated "Use OpenApiSpex.Plug.CastAndValidate.init/1 instead"
   def init(opts), do: Keyword.put_new(opts, :render_error, OpenApiSpex.Plug.DefaultRenderError)
 
   @impl Plug
+  @deprecated "Use OpenApiSpex.Plug.CastAndValidate.call/2 instead"
   def call(conn, render_error: render_error) do
     spec = conn.private.open_api_spex.spec
     operation_id = conn.private.open_api_spex.operation_id
     operation_lookup = conn.private.open_api_spex.operation_lookup
     operation = operation_lookup[operation_id]
-    content_type = Conn.get_req_header(conn, "content-type")
-        |> Enum.at(0, "")
-        |> String.split(";")
-        |> Enum.at(0)
+
+    content_type =
+      Conn.get_req_header(conn, "content-type")
+      |> Enum.at(0, "")
+      |> String.split(";")
+      |> Enum.at(0)
```

Comment:
I would extract that to separate helper function as this is used twice in different modules.

Reply:
I would probably use pattern matching instead with `with` instead of `Enum.at/{2,3}`.

Reply:
It probably doesn't make sense to make any changes to this module because it's being deprecated. These changes were added by Elixir Formatter.

Reply:
@moxley [this is the same here](https://github.com/open-api-spex/open_api_spex/pull/153/files#diff-979f2c7480c57b48639546044c527939R70)